packages/leancode_lint/README.md

```diff
@@ -413,6 +413,203 @@ class MyWidget extends StatelessWidget {
 
 None
 
+### `bloc_class_modifiers`
+
+**DO** add `final` or `sealed` modifiers to bloc state, event, and presentation event classes.
+
+**BAD:**
+
+```dart
+class MyState {}
+
+class MyStateInitial extends MyState {}
+
+class MyStateLoading extends MyState {}
+```
+
+**GOOD:**
+
+```dart
+sealed class MyState {}
+
+final class MyStateInitial extends MyState {}
+
+final class MyStateLoading extends MyState {}
+```
+
+#### Configuration
+
+None.
+
+### `bloc_const_constructors`
+
+**DO** define unnamed const constructors for bloc state, event, and presentation event classes.
```

Comment:
Ditto, it would be nice to highlight that this lint works only if bloc is in the same file as states, events etc